libtrip-notification/src/main/java/com/mapbox/navigation/trip/notification/NavigationNotificationProvider.kt

```diff
@@ -3,7 +3,21 @@ package com.mapbox.navigation.trip.notification
 import android.app.Notification
 import androidx.core.app.NotificationCompat
 
+/**
+ * Provide notification for navigation process launched with a [TripSession]
+ *
+ * @since 1.0.0
+ */
 object NavigationNotificationProvider {
```

Comment:
This class should be internal and doesn't require public docs. We still can include guidelines and docs for ourselves though :+1: